installing/installing_vsphere/installing-vsphere-post-installation-configuration.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="installing-vsphere-post-installation-configuration"]
+= Configuring the vSphere connection parameters using the web console
+include::_attributes/common-attributes.adoc[]
+:context: installing-vsphere-post-installation-configuration
+
+
+After installing an {product-title} cluster using the Assisted Installer on vSphere with the platform integration feature enabled, you must update the vSphere connection settings manually. This is because the Assisted Installer adds invalid connection settings to the *vSphere connection* wizard during the installation. 
```

Comment:
Instead of `invalid`, what about denoting the fact that they are placeholders? Logically `invalid` is correct, but sounds strong or as a bug to me.